axlearn/audio/asr_decoder_test.py

```diff
@@ -89,6 +91,97 @@ def test_map_label_sequences(
         )
 
 
+class ValidCtcSeqTest(TestCase):
+    def get_logits_and_labels(
+        self, batch_size: int, time_steps: int, target_steps: int, nclasses: int
+    ) -> tuple[Tensor, Tensor, Tensor, Tensor]:
+        prng_key = jax.random.PRNGKey(1234)
+        logits = jax.random.normal(prng_key, (batch_size, time_steps, nclasses), dtype=jnp.float32)
+        paddings = jnp.zeros((batch_size, time_steps), dtype=np.int32)
+        target_labels = jax.random.randint(
+            prng_key,
+            shape=(batch_size, target_steps),
+            minval=1,
+            maxval=nclasses - 1,
+            dtype=jnp.int32,
+        )
+        target_paddings = jnp.zeros(shape=(batch_size, target_steps), dtype=jnp.int32)
+        return logits, paddings, target_labels, target_paddings
+
+    def test_label_longer_than_input(self):
+        batchsize = 4
+        timesteps = 10
+        labelsteps = 11
+        nclasses = 400
+        # generate logits and labels, which has logits shorter than labels
```

Comment:
Nit: Use full sentence for comment: start with Capitalize, and end with period `.`